StortSpelProjekt/Game/src/Gamefiles/Components/EnemyComponent.cpp

```diff
@@ -66,7 +66,7 @@ void component::EnemyComponent::OnUnInitScene()
     tc->GetTransform()->SetPosition(parentPos.x, parentPos.y, parentPos.z);
 
     entity->SetEntityState(true);
```

Comment:
Now happens twice. This happens inside InitDynamicEntity. This function should be private now. And only be called from Scene::InitDynamicEntity